architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -11,11 +11,21 @@
 toc::[]
 
 == Builds
-A build is the process of transforming input parameters into a resulting object.
-Most often, the process is used to transform source code into a runnable image.
-A link:../../dev_guide/builds.html#defining-a-buildconfig[BuildConfig] object is
+A link:../../dev_guide/builds.html[build] is the resulting object of
```

Comment:
This seems like it's losing some of its meaning. We're going from saying a build is a process to now saying it's the resulting object of itself? In the original version, the resulting object was the runnable image.